### DISCUSSION TITLE
Hydration issues

### DISCUSSION THREAD

**Discussion by aplasa85** [neuefische/questions#350] [Web] [answered]
Hi there,

I am  currently encountering an hydration issue with my page and I could need some assistance. After adding the footer to my code, I started getting the following message:

"  Unhandled Runtime Error

Error: Hydration failed because the initial UI does not match what was rendered on the server.

See more info here: https://nextjs.org/docs/messages/react-hydration-error  "

Seemingly, I have created some sort of conflict here, but I couldn't tell which or where. The issue is happening on the content branch of  my repo. 

https://github.com/aplasa85/capstone-template-ffm-web-22-1/tree/content

Thank you for your help.

Axel

**Comment by stefankorder** [comment, marked as answer]
Hi Axel,

in from Line 14 to 29 in your index.js inside your pages folder your try to display the following:

```
<StyledParagraph>
<p>Welcome to the Casa
del Agave online shop. 

Here you can find a fine
selection of tequila and
mezcal brands delivered
right to your home.

Thank you for choosing us 
and we certainly hope 
you will find the beverage 
of your convenience.</p>


</StyledParagraph>
```

Both, the component and the "html-tag" are "p-tags". You can't wrap a p-tag inside another p-tag. You should change the first tag to a div or a section.
I hope I could help you.

**Reply by aplasa85** [reply]
Thanks a lot.
